R/caretList.R

```diff
@@ -255,6 +255,25 @@ is.caretList <- function(object){
   is(object, "caretList")
 }
 
+#' @title Coerce object to caretList object
+#' @description Coerces list of caret models into a caretList
+#' @param list of caret models
+#' @return a \code{\link{caretList}} object
+#' @export
```

Comment:
I tried this, but `as.caretList` doesn't get assigned as a S3 method to the caretList class, for whatever reason. It works as like another package function at the moment. I'm still trying to figure that one out, but it works :/

Since roxygen2 version 3.0+ it automatically does the work for finding S3 methods. If the `@export` is there. The only time you do need it, is when there is a collision and roxygen can't figure it out.

from the man himself:
[http://stackoverflow.com/questions/6517222/how-to-properly-document-a-s3-method-of-a-generic-from-a-different-package-usin](http://stackoverflow.com/questions/6517222/how-to-properly-document-a-s3-method-of-a-generic-from-a-different-package-usin)

However, I don't think there is a penalty for adding it, besides another line of code and perhaps the added risk of human error.